src/app/code/community/Zendesk/Zendesk/Helper/Data.php

```diff
@@ -292,6 +297,19 @@ public function getTicketUrl($row, $link = false)
         return '<a href="' . $url . '" target="_blank">' .  Mage::helper('core')->escapeHtml($subject) . '</a>';
     }
     
+    public function getRequestUrl($row, $link = false)
```

Comment:
@buheryfi This function looks a lot like getTicketUrl() , is there any way you can change it so that there wouldn't be too much repetition? 
Thanks for this!